packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -176,14 +175,40 @@ class RunPanel extends React.PureComponent {
                     {content}
                 </Drawer>
                 {is_mobile && <MobileDrawerFooter {...this.props} should_show_statistics={active_index !== 2} />}
-                <Dialog
+                <Modal
                     title={dialog_options.title}
                     is_open={is_dialog_open}
-                    onOkButtonClick={onOkButtonClick}
-                    onCancelButtonClick={onCancelButtonClick}
+                    toggleModal={onCancelButtonClick || onOkButtonClick}
```

Comment:
Can we create a toggle function instead? It'll be clearer and it seems that in some cases `onOkButtonClick` clears the statistics too.

Reply:
we have onCloseDialog action in run-panel-store 
I use that function to toggleModel